sources/HUBComponentWrapper.m

```diff
@@ -282,8 +281,9 @@ - (void)component:(id<HUBComponentWithChildren>)component didStopDisplayingChild
         return;
     }
     
+    HUBComponentWrapper * const childComponent = self.childrenByIndex[@(childIndex)];
```

Comment:
Oh damn, didn't know that! So would these UIView's also be able to conform to HUBComponentContentObserver, or is it alright if we only add support for content offset observing for the components created through the framework?